### ISSUE TITLE
task fails to load/ use external configuration file

### ISSUE THREAD

**Issue by karlgroves** [egauci/grunt-tenon-client#4]
Per the README file, this task can receive its options via an external file, however doing so causes the task to fail.

Steps to reproduce:
Configure the task in Gruntfile.js like so
```
tenon: {
  options: {
    config: '.tenonrc'
  }
  /* other task stuff here */
}
```

Expected result:  Configuration options set in this external file (correctly formatted as JSON) will be used when task is run

Actual result: Task fails to run, displaying error stating that key is required.

**Comment by egauci** [comment]
Hi @karlgroves  - I can't recreate this with a good '.tenonrc' file. However, if the JSON doesn't parse, this is what you get. It doesn't complain, it just proceeds without it.

That's probably a mistake. I think I may make it throw by default, with an option to ignore.

**Comment by karlgroves** [comment]
GAH. You're right. It was bad JSON.  It looked perfect except for one small silly issue.

Sorry!

**Comment by egauci** [comment]
hi @karlgroves - no worries -- JSON is picky. :) I did the exact same thing in my testing, and it was a JSON with just the one entry in it.

Anyway - I just released v0.7.0 that gives a proper error when this happens. Thanks for catching.
